bgpd/bgp_rpki.c

```diff
@@ -621,6 +631,25 @@ static struct rtr_mgr_group *get_connected_group(void)
 	return rtr_mgr_get_first_group(rtr_config);
 }
 
+static void print_prefix_table_by_asn(struct vty *vty, as_t as)
+{
+	struct rtr_mgr_group *group = get_connected_group();
+	struct rpki_for_each_record_arg arg;
+
+	arg.vty = vty;
+	arg.as = as;
+
+	if (!group) {
+		vty_out(vty, "Cannot find a connected group.\n");
+		return;
+	}
+
+	struct pfx_table *pfx_table = group->sockets[0]->pfx_table;
+
+	pfx_table_for_each_ipv4_record(pfx_table, print_record_by_asn, &arg);
+	pfx_table_for_each_ipv6_record(pfx_table, print_record_by_asn, &arg);
+}
```

Comment:
and dump the counted prefixes identified

Reply:
fixed